beakerx/setup.py

```diff
@@ -56,7 +56,15 @@ def update_package_data(distribution):
     # distribution.package_data = find_package_data()
     # re-init build_py options which load package_data
     build_py.finalize_options()
+    magic = distribution.get_command_obj('install_magic')
+    magic.run()
 
+class InstallMagic(Command):
+    def run(self):
+        c = get_config()
+        c.InteractiveShellApp.extensions = [
+            'groovy_magicaaa'
```

Comment:
fix typo. This code should add extension to config file. Then extension should be loaded during jupyter start up.